### DISCUSSION TITLE
Duda TP1

### DISCUSSION THREAD

**Discussion by Ramiringo** [INGCOM-UNRN-P1/cursada-2023#23] [Preguntas y respuestas] [answered]
Tengo una duda sobre el TP1, a que se refiere con esto: 

Indicar en un comentario la instrucción de compilación con GCC por línea de comandos.

**Comment by martinvilu** [comment]
Cual es la instrucción usada para compilar el archivo.

O estas usandolo solo desde vscode?

**Reply by Ramiringo** [reply]
lo estoy usando solo desde vscode

**Reply by martinvilu** [reply]
Bueno, eso

**Reply by Ramiringo** [reply]
que lo estoy compilando desde visual?

**Reply by martinvilu** [reply, marked as answer]
Yeap, igual si te fijas en la terminal de `vscode`, en algun lado se debe de leer algo parecido a `gcc -o miprograma.exe archivo.c`
